src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/event/CaseEventReceiver.java

```diff
@@ -11,8 +11,6 @@
 import uk.gov.ons.ctp.common.event.model.CaseEvent;
```

Comment:
Could we rename the 'eventreceiver' classes to accurately reflect what they are receiving please.
When we call RM via REST to get a case, we can optionally request that the case has RM CaseEvent[] populated - CaseEvents are events that RM store against a case.
Suggest 'CaseUpdateEventReceiver' and 'SurveyUpdateEventReceiver'

Reply:
Done